Jjamidei patch 1

JJAmidei opened the pull request:
@klieret @pibion 
Changed setup.md page to have installation instructions for windows, mac, and linux.

pibion committed:
add figures illustrating an ssh connection

JJAmidei committed:
Added initial text to Setup

JJAmidei committed:
Merge branch 'hsf-training:gh-pages' into gh-pages

JJAmidei committed:
Edited the text in setup.md and added a link for WSL installation.

JJAmidei committed:
Added information about navigating to terminal application for macOS and Linux as well as made general edits to setup.md.

JJAmidei committed:
Merge branch 'hsf-training:gh-pages' into new-branch-1

JJAmidei committed:
Changed formatting to try and emphasis key commands.

JJAmidei committed:
Merge branch 'new-branch-1' of github.com:JJAmidei/hsf-training-ssh-webpage into new-branch-1

JJAmidei committed:
Update setup.md

Added "Windows/Mac/Linux Installation and Access" sections. Also changed link for "How to use Terminal on Mac" article to the "Unix Shell" software carpentry setup page (i.e. where the link was originally found).

JJAmidei committed:
Update setup.md

Changed size of font for section titles

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

klieret commented on setup.md:
```diff
@@ -1,7 +1,25 @@
+
+### Windows Installation and Access
+
+If you are using Windows, you will need to install [the Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install). WSL will have the ssh programs you need by default, so once you've installed WSL you are ready for this tutorial.
```
Note that newer Windows systems have SSH pre-installed in the Windows shell and you can use it in the same way. However, I'm not sure how it behaves regarding configs and the like. But figuring that out and perhaps adding a note could be something to figure out.

JJAmidei replied:
Noted. Will look into and add whatever is relevant.

klieret commented on fig/ssh_connection.png:
Are you planning to use this picture somewhere else? Because it doesn't seem to be referenced in the `setup.md`...

JJAmidei replied:
No, I don´t think so. I'll double check with Amy, but I'm fairly certain that she made it for this tutorial specifically. I think she intended for it to be used on the Introduction page.

klieret reviewed (approved):
Thanks a lot for this PR. Much appreciated!

JJAmidei committed:
Update setup.md

Added a brief section about OpenSSH for newer versions of windows as well as a link to Microsoft's page on getting started with OpenSSH.

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

klieret commented on setup.md:
```diff
@@ -1,7 +1,27 @@
+
+If you are using a version of Windows, you will need to install [the Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install). WSL will have the ssh programs you need by default, so once you've installed WSL you are ready for this tutorial.
+
+Note: If your version of Windows is either Windows 10 version 1809 or newer, then you have access to a windows ssh program called OpenSSH. More information about OpenSSH can be found [here](https://learn.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse?tabs=gui).
```
This should come first. If people have Win > 10, they don't need WSL, right (it currently seems to contradict the first sentence)?

JJAmidei replied:
You're right. I'll go ahead and change that.

JJAmidei committed:
Update setup.md

Changed the order of instructions in the "Windows Installation and Access" section + grammar

pre-commit-ci[bot] committed:
[pre-commit.ci] pre-commit autoupdate

updates:
- [github.com/codespell-project/codespell: v2.3.0 → v2.4.1](https://github.com/codespell-project/codespell/compare/v2.3.0...v2.4.1)

pibion committed:
Merge branch 'gh-pages' into jjamidei-patch-1

pibion committed:
Merge branch 'gh-pages' into jjamidei-patch-1

faganso committed:
Moving complex material to appendix

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

faganso committed:
Appendix

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

faganso committed:
move nested material later

faganso committed:
Merge branch 'jjamidei-patch-1' of https://github.com/JJAmidei/hsf-training-ssh-webpage into jjamidei-patch-1

faganso committed:
Create 03-config.md

faganso committed:
Intro

faganso committed:
Updated Keberos Info

faganso committed:
first edits

faganso committed:
Merge branch 'pre-commit-ci-update-config' into jjamidei-patch-1

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

faganso committed:
finished draft of Kerberos section and added to appendix

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

faganso committed:
Update 01-introduction.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

faganso committed:
Update 01-introduction.md

faganso committed:
Merge branch 'jjamidei-patch-1' of https://github.com/JJAmidei/hsf-training-ssh-webpage into jjamidei-patch-1

faganso committed:
Update 01-introduction.md

faganso committed:
Added information on required software

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

faganso committed:
Added a question and a warning to keys

faganso committed:
Merge branch 'jjamidei-patch-1' of https://github.com/JJAmidei/hsf-training-ssh-webpage into jjamidei-patch-1

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

faganso committed:
Renamed files

faganso committed:
Merge branch 'jjamidei-patch-1' of https://github.com/JJAmidei/hsf-training-ssh-webpage into jjamidei-patch-1

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

faganso committed:
Reworded Keys and Kerberos

faganso committed:
Merge branch 'jjamidei-patch-1' of https://github.com/JJAmidei/hsf-training-ssh-webpage into jjamidei-patch-1

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci
